crates/hyperswitch_domain_models/src/customer.rs

```diff
@@ -36,7 +35,6 @@ pub struct Customer {
     pub address_id: Option<String>,
     pub default_payment_method_id: Option<String>,
     pub updated_by: Option<String>,
-    pub version: ApiVersion,
```

Comment:
why are these deleted, please add them back

Reply:
The idea is to have a single place to set this value, and this should be set at the database layer only. But since we might need this in logs in the future, will add this back